.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         target:
           - x64
         node:
-          - 18
```

Comment:
I don't think there'll be much trouble if any at all. I did this because latest electron-builder requires Node v20+. No other changes were needed and it just worked.

I have been using v22.6.0 for development. From my experience, Desktop worked perfectly with v22, so you might even want to try that.